.github/workflows/publish-to-npm.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish to NPM
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm install
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
can you share this secret NPM_TOKEN with a couple of other folks so we have some redundancy?

Reply:
Yeah, I was just waiting for the final decision on the NPM org name in our Slack conversation. Will share on Keybase once we finalise.